Dockerfile

```diff
@@ -63,4 +63,4 @@ COPY tests/data/hsc_raw hsc_raw
 
 EXPOSE 8080
 
-ENTRYPOINT ["exposurelog", "run", "--port", "8080"]
+ENTRYPOINT ["uvicorn", "exposurelog.app:app", "--port", "8080"]
```

Comment:
I have updated it, closely following the linked version. A few questions:
* My Dockerfile creates a non-root user. I'm guessing that's still a good idea, but gafaelfawr doesn't have it. Should I remove it?
* Do I still need `EXPOSE 8080`? I took it out because gafaelfawr doesn't have it.

Reply:
Creating a non-root user is probably better since it means the right thing happens when the image is run directly via Docker. I didn't bother with Gafaelfawr since I only support running it via the Helm chart and the Helm chart runs the Docker image as a non-root user anyway so the Docker user machinery isn't used. But what you have seems fine.

It doesn't look like you need `EXPOSE 8080` (Kubernetes seems to figure it out anyway) but I think it's more correct to have it, and I should go back and add it to Gafaelfawr. Or alternately you could just go with the upstream default and have the server run on port 80 and change the chart, which is probably what I should do in Gafaelfawr (I didn't because I wanted to get the 2.0 release out).

Reply:
Using the upstream default sounds appealing. Just to be sure I understand:
* Get rid of ``ENV PORT=8080``
* Continue to leave out ``EXPOSE <port>``

If I do that then where is the port configured? I'd like to update the README accordingly.

Reply:
Yes, in that case the application will be started on port 80 (so the Helm chart would need to be updated to match to register the correct service). It's documented here:

https://github.com/tiangolo/uvicorn-gunicorn-docker#port